packages/core/admin/admin/src/pages/ProfilePage/index.js

```diff
@@ -495,7 +495,7 @@ const ProfilePage = () => {
                                 id: 'Settings.profile.form.section.experience.clear.select',
                                 defaultMessage: 'Clear the interface language selected',
                               })}
-                              value={values.preferedLanguage}
+                              value={values.preferedLanguage || ''}
```

Comment:
This shouldn't be needed anymore, right @joshuaellis?

Reply:
ah ok, is it resolved then?

Reply:
yes now it works